homeassistant/components/zha/api.py

```diff
@@ -340,15 +232,15 @@ def async_load_api(hass, application_controller, listener):
     async def websocket_entities_by_ieee(hass, connection, msg):
         """Return a dict of all zha entities grouped by ieee."""
         entities_by_ieee = {}
-        for ieee, entities in listener.device_registry.items():
+        for ieee, entities in zha_gateway.device_registry.items():
             ieee_string = str(ieee)
             entities_by_ieee[ieee_string] = []
             for entity in entities:
-                if not isinstance(entity, ZhaDeviceEntity):
-                    entities_by_ieee[ieee_string].append({
-                        ATTR_ENTITY_ID: entity.entity_id,
-                        DEVICE_INFO: entity.device_info
-                    })
+                entities_by_ieee[ieee_string].append({
+                    ATTR_ENTITY_ID: entity.reference_id,
```

Comment:
Also here I'd like us to rename `entity` to eg `entity_ref`.

Reply:
This PR or in a quick followup?